examples/dashboard.json

```diff
@@ -671,7 +671,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "sum(rollout_phase{phase=\"Paused\"} == 1)",
+          "expr": "sum(rollout_phase{phase=\"Error\"} == 1)",
```

Comment:
I think you may need to rebase your changes. This seems to be reflecting a change which already got merged:
https://github.com/argoproj/argo-rollouts/commit/99bd90e5c8987fd8e8c1ddcff818ad82680130db